liquibase/pom.xml

```diff
@@ -0,0 +1,89 @@
+<!--
+
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+    graphic logo is a trademark of OpenMRS Inc.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<parent>
+		<groupId>org.openmrs</groupId>
+		<artifactId>openmrs</artifactId>
+		<version>2.4.0-SNAPSHOT</version>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.openmrs.liquibase</groupId>
+	<artifactId>openmrs-liquibase</artifactId>
+	<name>openmrs-liquibase</name>
+	<description>Generate  running liquibase change sets from the console</description>
+
+	<dependencies>
+		<dependency>
+			<groupId>mysql</groupId>
+			<artifactId>mysql-connector-java</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.azeckoski</groupId>
+			<artifactId>reflectutils</artifactId>
+			<version>0.9.20</version>
+		</dependency>
+		<dependency>
+			<groupId>org.openmrs.api</groupId>
+			<artifactId>openmrs-api</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.openmrs.liquibase.ext</groupId>
+			<artifactId>identity-insert</artifactId>
+			<version>1.2.1</version>
+		</dependency>
+		<dependency>
+			<groupId>org.openmrs.liquibase.ext</groupId>
+			<artifactId>modify-column</artifactId>
```

Comment:
Didn't we eventually copy over code from the dependency directly to openmrs core?

Reply:
You are right, we had copied three classes to the codebase of openmrs-api. However, openmrs-liquibase does not depend on that codebase but is a self contained module for generating snapshots. Hence it continues to rely on the library to get those classes.

Reply:
I think it would be better to refactor those three classes into a self-contained jar depended on by OpenMRS API and the Liquibase part then. At least that way they're sharing the same classes!

Reply:
`openmrs-liquibase` does not need the Liquibase extensions and they were removed from the pom file.